package-overrides/github/angular/bower-angular-resource@1.4.8.json

```diff
@@ -0,0 +1,13 @@
+{
+  "format": "cjs",
```

Comment:
Only global format works with shims currently, so it may be best to leave this out.